cli/pull.ml

```diff
@@ -136,7 +125,11 @@ let run yes repo () =
       let duniverse_dir = Fpath.(repo // Config.vendor_dir) in
       Bos.OS.Dir.create duniverse_dir >>= fun _created ->
       mark_duniverse_content_as_vendored ~duniverse_dir >>= fun () ->
-      pull_source_dependencies ~duniverse_dir duniverse
+      get_cache ~no_cache >>= fun cache -> pull_source_dependencies ~duniverse_dir ~cache duniverse
+
+let no_cache =
+  let doc = "Run without using the duniverse global cache" in
```

Comment:
Before merging, could the help be improved so that the actual behaviour is described here? In particular, the environment variables that are looked up so that user's know what's going on and how to modify it.

Reply:
Yeah really good point! I'll improve the man page!

Reply:
I added some documentation about that, let me know what you think!